src/vivarium_cluster_tools/psimulate/redis_dbs/registry.py

```diff
@@ -300,5 +301,9 @@ def update_and_report(self) -> Dict[str, Union[int, float]]:
         self._logger.info(template.format(**status))
         return status
 
+    def get_params_by_job(self):
```

Comment:
type hint the return

Reply:
I'm never sure how far to go down the line with nested dictionaries outside of the layeredconfigtree context.
Should I do dict[str, dict] or try to go down to dict[str, dict[str, Any] or dict[str, dict[str, Union[str, int, dict]]??